backend-old/ohq/settings.py

```diff
@@ -11,11 +11,11 @@
 """
 
 import os
-import django_heroku
+
 import dj_database_url
+# import django_heroku
```

Comment:
I personally prefer to just delete code snippets like this instead of leaving them as comments, since everything is stored in Git and comments like these tend to clutter the code base over time.

Up to you guys how you want to handle this, but code comments like these usually stay in a code base for too long because they're not cleaned up.

Reply:
I see that the entire `backend-old` folder is unused, I'd prefer just deleting this and having it in Git history instead of keeping it around.